### DISCUSSION TITLE
Wrong lastRan timestamp

### DISCUSSION THREAD

**Discussion by michele-grifa** [spatie/laravel-health#47] [Q&A]
Hi, i'm trying the package but when i use and open the health page, it shows me a wrong last ran (8 months from now).
I checked the entries in the database and the timestamp is right.

**![bug](https://user-images.githubusercontent.com/33720244/148808515-334accfa-0351-4d3d-bb82-601c097af5a0.png)**

**Comment by freekmurze** [comment]
I cannot seem to reproduce this. Should you find that the package is in error, send in a PR to fix it.

**Reply by michele-grifa** [reply]
After a bit of testing i found the problem, in the migration the timestamps have precision 0 and on SQL Server 2017 with IT Locale, cause the inversion of day and month. Setting precision to 2 resolve the problem for me, but i don't know if is a general solution.
